WooCommerce/UITestsFoundation/XCTest+Extensions.swift

```diff
@@ -47,18 +47,6 @@ extension XCTestCase {
         }
     }
 
-    public func waitForElementToNotExist(element: XCUIElement, timeout: TimeInterval? = nil) {
-        let notExistsPredicate = NSPredicate(format: "exists == false")
-        let expectation = XCTNSPredicateExpectation(predicate: notExistsPredicate,
-                                                    object: element)
-
-        let timeoutValue = timeout ?? 30
-        guard XCTWaiter().wait(for: [expectation], timeout: timeoutValue) == .completed else {
-            XCTFail("\(element) still exists after \(timeoutValue) seconds.")
-            return
-        }
-    }
-
```

Comment:
moved this to the correct extension, previously this was under `XCTestCase` which was odd when calling it (though i end up not using this in the end, i can remove this change if it makes it better)